index.d.ts

```diff
@@ -39,23 +39,24 @@ declare module 'leaflet' {
 
         export class Sidebar extends L.Control { 
             constructor(options?: SidebarOptions);
+        }
+    }
 
-            addTo(map: L.Map): this;
-            removeFrom(map: L.Map): this;
-
-            open(id: string): this;
-            close(): this;
+    interface Control {
```

Comment:
Why are these functions not class methods of `Sidebar` anymore? doesnt look right. Although it's been a while that I last typescripted..

Reply:
I also don't have a working test environment atm where I could verify the types :(

Reply:
TL;DR : You should decline this PR!

Yeah - I had accidentally asserted that the control type was Control and not Control.Sidebar. If you correctly assert the type of the control - the problem I had goes away. I pressed the PR trigger too quickly and I was right first time.